kazoo/tests/test_watchers.py

```diff
@@ -418,6 +418,40 @@ def changed(children):
         update.wait(0.5)
         eq_(all_children, ['smith'])
 
+    def test_remove_session_watcher_on_func_stops(self):
+        update = threading.Event()
+        all_children = ['fred']
+
+        fail_through = []
+
+        def changed(children):
+            while all_children:
+                all_children.pop()
+            all_children.extend(children)
+            update.set()
+            if fail_through:
+                return False
+
+        childrenWatch = self.client.ChildrenWatch(self.path, changed)
```

Comment:
Maybe it's better to keep a snake case notation for this variable?

Reply:
Thank you for your comment. I use java in most time and carelessly write the variable with camel case .  I will turn it into snake case notation.